<launch>
    <include file="$(find zed_wrapper)/launch/zed2i.launch"/>
    <node name="pcl_filter" pkg="sensor_filters" type="pointcloud2_filter_chain" output="screen">
        <rosparam command="load" file="$(find idmp_ros)/config/zedFilterChain.yaml" />
        <remap from="~input" to="/zed2i/zed_node/point_cloud/cloud_registered" />
        <remap from="~output" to="/points_filt" />
    </node>
    
    <include file="$(find aruco-tools)/launch/publishTags.launch">
      <arg name="yaml_file"         value="$(find aruco-tools)/config/poses_kuka.yaml" />
    </include>    
    
    <rosparam file="$(find idmp_ros)/config/params_live.yaml" />
    <node pkg="idmp_ros" type="idmp" name="idmp" output="screen" respawn="true"/> 

</launch>